spec/services/user_destroyer_spec.rb

```diff
@@ -45,6 +45,11 @@
         StaffActionLogger.any_instance.expects(:log_user_deletion).with(@user, anything).once
         destroy
       end
+
+      it "shouldn't log the action if quiet is true" do
+        StaffActionLogger.any_instance.expects(:log_user_deletion).with(@user, anything).never
```

Comment:
I see the surrounding code using stubs but we're trying to move away from it :grin: 

We can just do something like 

```
expect {}.to_not change { UserHistory.count } 
```